dwave_networkx/drawing/chimera_layout.py

```diff
@@ -230,87 +229,86 @@ def draw_chimera(G, **kwargs):
-        A dict of chains associated with each node in G.  Should be
-        of the form {node: chain, ...}.  Chains should be iterables
-        of qubit labels (qubits are nodes in G).
+        Embedding ``emb`` for all nodes of ``G`` as a dict of chains
+        of the form ``{node: chain, ...}``. where chains are iterables
+        of qubit labels. Qubits are nodes in ``G``.
 
     embedded_graph : NetworkX graph (optional, default None)
-        A graph which contains all keys of emb as nodes.  If specified,
-        edges of G will be considered interactions if and only if they
-        exist between two chains of emb if their keys are connected by
-        an edge in embedded_graph
+        Graph which contains all keys of the ``emb`` parameter as nodes. If specified,
+        the edges of ``G`` will be considered interactions if and only if they
+        exist between two chains of the emb parameter and if their keys are connected by
+        an edge in the ``embedded_graph`` parameter; only the couplers between
+        chains with intended couplings are displayed.
 
     interaction_edges : list (optional, default None)
-        A list of edges which will be used as interactions.
+        Interactions as a list of edges.
+        If this parameter is specified, only the couplers in the list are displayed.
 
     show_labels: boolean (optional, default False)
-        If show_labels is True, then each chain in emb is labelled with its key.
+        If True, each chain in the ``emb`` parameter is labelled with its key.
 
     chain_color : dict (optional, default None)
-        A dict of colors associated with each key in emb.  Should be
-        of the form {node: rgba_color, ...}.  Colors should be length-4
-        tuples of floats between 0 and 1 inclusive. If chain_color is None,
+        Chain colors associated with each key in the ``emb`` parameter as a dict
+        of the form ``{node: rgba_color, ...}``, where colors must be length-4
+        tuples of floats between 0 and 1, inclusive. If None,
         each chain will be assigned a different color.
 
     unused_color : tuple (optional, default (0.9,0.9,0.9,1.0))
-        The color to use for nodes and edges of G which are not involved
-        in chains, and edges which are neither chain edges nor interactions.
-        If unused_color is None, these nodes and edges will not be shown at all.
+        Color to use for graph ``G``'s nodes and edges that are not part of
+        chains, and edges that are neither chain edges nor interactions.
+        If None, these nodes and edges will not be shown.
 
     overlapped_embedding: boolean (optional, default False)
-        If overlapped_embedding is True, then chains in emb may overlap (contain
-        the same vertices in G), and the drawing will display these overlaps as
+        If True, chains in the ``emb`` parameter may overlap (contain
+        the same vertices in ``G``), and the drawing will display these overlaps as
```

Comment:
"the drawing will display these overlaps" --> "the drawing displays these overlaps"